infra/resources/prod/_modules/custom_roles/cosmos_query.tf

```diff
@@ -0,0 +1,13 @@
+resource "azurerm_role_definition" "cosmos_query" {
+  name        = "CosmosDB Data Operator"
```

Comment:
with this resource we define this `CosmosDB Data Operator` role, but how it is assigned to our web applications?